collectors/python.d.plugin/anomalies/README.md

```diff
@@ -206,7 +206,7 @@ If you would like to go deeper on what exactly the anomalies collector is doing
+- `lags_n`, `smooth_n`, and `diffs_n` together define the preprocessing done to the raw data before models are trained and before each prediction. This essentially creates a [feature vector](https://en.wikipedia.org/wiki/Feature_(machine_learning)#:~:text=In%20pattern%20recognition%20and%20machine,features%20that%20represent%20some%20object.&text=Feature%20vectors%20are%20often%20combined,score%20for%20making%20a%20prediction.) for each chart model (or each custom model). The default settings for these parameters aim to create a rolling matrix of recent smoothed [differenced](https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average#Differencing) values for each chart. The aim of the model then is to score how unusual this 'matrix' of features is for each chart based on what it has learned as 'normal' from the training data. So as opposed to just looking at the single most recent value of a dimension and considering how strange it is, this approach looks at a recent smoothed window of all dimensions for a chart (or dimensions in a custom model) and asks how unusual the data as a whole looks. This should be more flexible in capturing a wider range of [anomaly types](https://andrewm4894.com/2020/10/19/different-types-of-time-series-anomalies/) and be somewhat more robust to temporary 'spikes' in the data that tend to always be happening somewhere in your metrics but often are not the most important type of anomaly (this is all covered in a lot more detail in the [deepdive tutorial](https://nbviewer.jupyter.org/github/netdata/community/blob/main/netdata-agent-api/netdata-pandas/anomalies_collector_deepdive.ipynb)).
```

Comment:
Can't see the changed word here—not sure if it's GitHub's UI or some other issue. Any way to pick it out?

Reply:
flexibile => flexible 🤓

Reply:
How?!? Your proficiency with all things Git continues to amaze...

Reply:
`git diff --word-diff` is your friend for some of this fwiw (if you have a local repo and are willing to pull in the branch -- there's magic for being able to do that).

Also, until I squash, you can look at the log for the file and see the commit, on average the commit message will at least give you the replacement string:

1. [View file]( https://github.com/netdata/netdata/blob/6a6c2bb33255ad9b919cd7bfba246160c8ba12ad/collectors/python.d.plugin/anomalies/README.md)
2. [History]( https://github.com/netdata/netdata/commits/59c30f63ae66262e3c8dcbf66aa8b308b7accfea/collectors/python.d.plugin/anomalies/README.md)
3. From here, you can just check each commit
4. Or, use [Blame]( https://github.com/netdata/netdata/blame/6a6c2bb33255ad9b919cd7bfba246160c8ba12ad/collectors/python.d.plugin/anomalies/README.md)
5. And scroll to the [line]( https://github.com/netdata/netdata/blame/6a6c2bb33255ad9b919cd7bfba246160c8ba12ad/collectors/python.d.plugin/anomalies/README.md#L209)
6. In this case, the UI clearly hints about the most recent change. You can then use the "[previous change](https://github.com/netdata/netdata/blame/aa8cf9aef6c4fac9a6520b5e9c8a7ae3ff0f4d78/collectors/python.d.plugin/anomalies/README.md)" thing in case you think there are multiple changes for the line.
7. Scrolling to the [same line]( https://github.com/netdata/netdata/blame/aa8cf9aef6c4fac9a6520b5e9c8a7ae3ff0f4d78/collectors/python.d.plugin/anomalies/README.md#L209) shows that it's unchanged from its surroundings.